public/locales/de/app.ftl

```diff
@@ -0,0 +1,7 @@
+support = Unterstützung
+
+feedback = Feedback geben
+
+terms = Allgemeine&nbsp;Geschäftsbedingungen&nbsp;und&nbsp;Datenschutzerklärung
+
+copyright = Teile dieses Inhalts sind © 1998-2018 von einzelnen mozilla.org-Mitwirkenden. Inhalt verfügbar unter einer Creative Commons-Lizenz
```

Comment:
/cc @flodolo @stasm

Do either of you have any guidance or suggestions on bundling the .ftl files directly in the source repo, versus doing something like a separate "blurts-server-l10n" repo for translations?

Reply:
Reasons to use a different repository:
* History: Pontoon will write in the repository every 20 minutes, once for each locale (if there are changes). History is going to become messy very quickly.
* Decouplement: You can develop on master and decide when strings are ready for translation.

Cons:
* You need scripts to import strings back.
* You need to expose strings in the l10n repository via PR.

Alternatively, we can also write in a branch instead of master, but then you need to deal with merges back and forth.

Side note: we'll need a dev server updated frequently for l10n testing. For that it might be easier to have the strings in the main repository.



From the point of view of configuration, if you want Pontoon to write here we just need to add its bot.

Reply:
@flodolo can you point us to examples of each kind of setup? I.e., same-repo vs. separate-repo?

Reply:
The one separate-repo I can remember off the top of my head is FxA's content server:

https://github.com/mozilla/fxa-content-server
vs
https://github.com/mozilla/fxa-content-server-l10n

I think the trick was putting a package.json in the [*-l10n repo](https://github.com/mozilla/fxa-content-server-l10n/blob/master/package.json) so you can include it as a dependency in your main repo's package.json. But of course, as I went to prove that, I notice that FxA team is using some Bash script[1] to download locales, so we may want to sync up with them to see why they switched to doing it that way (or maybe I'm misremembering how it was done originally).

[1] https://github.com/mozilla/fxa-content-server/blob/master/scripts/download_l10n.sh